client/consensus/manual-seal/src/consensus/aura.rs

```diff
@@ -67,8 +67,10 @@ where
 		+ UsageProvider<B>
 		+ ProvideRuntimeApi<B>,
 	C::Api: AuraApi<B, AuthorityId>,
+	P: Send + Sync + 'static,
```

Comment:
```suggestion
	P: Send + Sync,
```
Lifetime bound not strictly required here